elide-core/src/test/java/com/yahoo/elide/books/Author.java

```diff
@@ -93,6 +89,17 @@ public void setPenName(Pseudonym penName) {
         this.penName = penName;
     }
 
+    // TODO: Eventually we should support multiple argument computed attributes to be used as API _functions_
+    @ComputedAttribute
+    @Transient
+    public String getSpecialOrEcho() {
```

Comment:
Minor comment but I think it makes the tests more readable.  We have a lot of classes like Left, Right and FunWithComputedAttributes - basically meaningless things that we test.  

I think our model for graphQL tests should make sense.  Maybe this should be a computed attribute like fullName or something along those lines.

Reply:
Good idea